### PR TITLE
feat(navigation-menu): apply new specifications - INNO-939

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Display links of navigation menu in 2 to 4 columns.
Apply new conventions

reference: https://webgate.ec.europa.eu/CITnet/confluence/pages/viewpage.action?pageId=675479860

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 7f87eb0431e96c8851d133323cf2e1e788a294be

https://deploy-preview-610--europa-component-library.netlify.com

File: src/generic/generic-component-navigation-menu/generic-component-navigation-menu.twig [hunk 1]
```diff
@@ -57,12 +57,21 @@
...
-                {% for _child_link in _link.children_links %}
-                  {% set _child_link_class_name = 'ecl-navigation-menu__link' ~ (_child_link.is_active ? ' ecl-navigation-menu__link--active' : '') %}
-                  <li class="ecl-navigation-menu__item"><a href="{{ _child_link.href }}" class="{{ _child_link_class_name }}">{{ _child_link.label }}</a></li>
+              <div class="ecl-navigation-menu__group ecl-row" id="nav-menu-expandable-group-{{ loop.index }}">
+                {% for _column in _link.children_links %}
+                  <div class="ecl-col">
+                    {% if _column.title is defined and _column.title is not empty %}
+                    <div class="ecl-navigation-menu__title">{{ _column.title }}</div>
+                    {% endif %}
+                    <ul class="ecl-list--unstyled">
```
Comment (yhuard) [inline src/generic/generic-component-navigation-menu/generic-component-navigation-menu.twig]: `ecl-list--unstyled` without `ecl-list`?

File: src/systems/ec/ec-component/ec-component-navigation/ec-component-navigation-menu/ec-component-navigation-menu.scss [hunk 2]
```diff
@@ -1,334 +1,17 @@
...
-
-  .ecl-navigation-menu__item--active
-    > .ecl-navigation-menu__link[aria-expanded='true'] {
-    color: map-get($ecl-colors, 'blue-120');
-  }
-}
+@import '@ecl/generic-component-navigation-menu/generic-component-navigation-menu';
+
+@include ecl-navigation-menu(
+  $ecl-navigation-menu-bar-bg-mobile: map-get($ecl-colors, 'blue-75'),
```
Comment (yhuard) [inline src/systems/ec/ec-component/ec-component-navigation/ec-component-navigation-menu/ec-component-navigation-menu.scss]: `$ecl-navigation-menu-bar-bg-mobile: map-get($ecl-colors, 'blue-75'),` vs `$bar-bg-mobile: map-get($ecl-colors, 'blue-75'),`?

**Review by yhuard — COMMENTED** [review]
A few questions + one more here: why has the font-size changed from `font-size: 0.875rem;` to `font-size: 1rem;`?

File: src/generic/generic-component-navigation-menu/generic-component-navigation-menu.twig (hunk 1 above)
Comment (emeryro) [reply src/generic/generic-component-navigation-menu/generic-component-navigation-menu.twig]: right, I'll add it

**Comment by emeryro** [comment]
font sizes are the one described in specifications

File: src/systems/ec/ec-component/ec-component-navigation/ec-component-navigation-menu/ec-component-navigation-menu.scss (hunk 2 above)
Comment (emeryro) [reply src/systems/ec/ec-component/ec-component-navigation/ec-component-navigation-menu/ec-component-navigation-menu.scss]: to be discussed...

**Review by yhuard — APPROVED** [review]
Looks great!

**Pull request merged**